components/ScrollView/ScrollDriverProvider.js

```diff
@@ -22,6 +22,7 @@ export class ScrollDriverProvider extends PureComponent {
   static propTypes = {
     children: PropTypes.node,
     driver: DriverShape,
+    onAnimationDriverChange: PropTypes.func,
```

Comment:
let's document this prop and explain why we're using it. ( We need this prop to propagate the driver change to recalculate navigational styles. We won't have the need for this prop once we rewrite using the new Context API )